### PR TITLE
[#989] Fix GKE service subnets collision

### PR DESCRIPTION
**Pull request by karbyshevds**
Add `service_cidr` variable (must be part of VPC subnet) and use it to define GKE services CIDR to avoid subnet collisions.    

NOTE: there is also appropriate legion-profiles PR!

### CODE REVIEW

**Review by mcdoker18 — APPROVED** [review]

**Comment by kirillmakhonin** [comment]
Approved

**Comment by mcdoker18** [comment]
@karbyshevds  rebase please

File: terraform/env_types/gcp/gke/gke_create/variables.tf [hunk 1]
```diff
@@ -64,6 +64,10 @@ variable "pods_cidr" {
   description = "GKE pods CIDR"
 }
 
+variable "service_cidr" {
+  description = "GKE service CIDR"
+}
```
Comment (maxkochubey) [inline terraform/env_types/gcp/gke/gke_create/variables.tf:69]: Maybe we should add a default value?

**Review by maxkochubey — APPROVED** [review]

**Comment by karbyshevds** [comment]
> @karbyshevds rebase please

done

File: terraform/env_types/gcp/gke/gke_create/variables.tf (hunk 1 above)
Comment (karbyshevds) [reply terraform/env_types/gcp/gke/gke_create/variables.tf:69]: ideally - yes, practically - it has to be part of cluster VPC subnet which is unknown for this module.

**Review by kirillmakhonin — APPROVED** [review]

**Pull request merged**